coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 68.89
+    "covered_percent": 68.88
```

Comment:
🔕  I'm not sure who/what is consuming or using this but I usually ignore my changes to .last_run. I think if it's no longer consumed we could probably drop it from source control

Reply:
I'll add a note to ask about it in the MM meeting - I 100% feel it isn't something that should be a part of source control